inc/Engine/Admin/Settings/Render.php

```diff
@@ -286,6 +287,23 @@ public function textarea( $args ) {
 		echo $this->generate( 'fields/textarea', $args ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Dynamic content is properly escaped in the view.
 	}
 
+	/**
+	 * Displays the textarea field template.
+	 *
+	 * @since 3.0
+	 *
+	 * @param array $args Array of arguments to populate the template.
+	 */
+	public function custom_textarea( $args ) {
```

Comment:
I don't like this method name so much, how is it custom? Can it be re-used, or is it very specific to the delay JS?

Reply:
it may be confusing but this textarea is really not matching the remaining textareas in settings, check below:

Custom Textarea:
![image](https://user-images.githubusercontent.com/15707971/210332650-6ee67194-3127-422c-98ef-9c41bef2c742.png)

Normal Textarea:
![image](https://user-images.githubusercontent.com/15707971/210332570-4cbc2a91-1814-4891-bfc7-39a86f93b8e0.png)

So I'm not sure if in the future we will change the other ones to be exactly like this custom one or what.
But for now it's only for delay JS.

What do u think/suggest?

Reply:
maybe `user_exclusions_textarea()` ?

Reply:
Done, Many thanks.